resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSource.scala

```diff
@@ -17,25 +17,170 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
-import com.codahale.metrics.{Gauge, MetricRegistry}
+import java.util.concurrent.TimeUnit
+import java.util.Date
```

Comment:
This causes a scala style error:
java.util.Date is in wrong order relative to java.util.concurrent.TimeUnit

Reply:
Reordered